### DISCUSSION TITLE
Indian time zone problem

### DISCUSSION THREAD

**Discussion by woshishitou** [apache/skywalking#7834] [Q&A]
Our application services, skywalking agent/oap/webapp and elasticsearch are all deployed on servers in India.
Our Skywalking version is 8.4.
We did not configure any timezone configuration for skywalking agent/oap/webapp and elasticsearch.

The log time of skywalking and elasticsearch are both in the normal Indian timezone.
However, a problem was found in the track page of SkyWalking when it was queried by time.
The time zone setting in the lower right corner of the page, I set it to 5.5, but it still seems to be considered to be 5, and 5.5 does not take effect.

What should I do?
Thanks!

**Comment by woshishitou** [comment]
![image](https://user-images.githubusercontent.com/22254487/135214370-308130f0-7e91-43a6-ad7f-7bda547ca590.png)

**Comment by wu-sheng** [comment]
I think UI logic should be enhanced, it doesn't support float(int only) for now.
Could you try to contribute this?

**Reply by woshishitou** [reply]
I'd love to, but unfortunately I don't know front-end technology
